tests/ui/const-generics/issues/issue-67945-1.full.stderr

```diff
@@ -11,6 +11,7 @@ LL |         let x: S = MaybeUninit::uninit();
    |
    = note: expected type parameter `S`
                        found union `MaybeUninit<_>`
+   = note: the caller chooses a type for `S` which can be different from `MaybeUninit<_>`
```

Comment:
“caller” isn't accurate here, there's no function. While we could say something like “user of `Bug`” I'm not really sure we want to emit this note at all. See my other review comment.

Reply:
Moved the emission so it no longer notes for generic expression coercion.